packages/devtools_app/lib/src/shared/table/table.dart

```diff
@@ -1124,27 +1124,35 @@ class _TableState<T> extends State<_Table<T>> with AutoDisposeMixin {
                 const ThickDivider(),
               ],
               Expanded(
-                child: Scrollbar(
-                  thumbVisibility: true,
-                  controller: scrollController,
-                  child: GestureDetector(
-                    behavior: HitTestBehavior.translucent,
-                    onTapDown: (a) => widget.focusNode?.requestFocus(),
-                    child: Focus(
-                      autofocus: true,
-                      onKey: (_, event) => widget.handleKeyEvent != null
-                          ? widget.handleKeyEvent!(
-                              event,
-                              scrollController,
-                              constraints,
-                            )
-                          : KeyEventResult.ignored,
-                      focusNode: widget.focusNode,
-                      child: ListView.builder(
-                        controller: scrollController,
-                        itemCount: _data.length + widget.fillerRowCount,
-                        itemExtent: widget.rowItemExtent,
-                        itemBuilder: _buildItem,
+                child: LayoutBuilder(
```

Comment:
To be a bit more careful about introducing performance regressions, let's only add this layout builder when `widget.fillWithEmptyRows == true`.

Reply:
There is already a  LayoutBuilder in Table, updated to re-use the existing one to avoid introducing performance regression